common/json/impl/boost/boost_json.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
Not sure if we want to care about it now, but `#pragma`s are generally not very portable.

Reply:
I don't remember if we discussed this in the first meeting... If you have experience of not being portable we should switch to header guards as soon as possible.

I'll do this :memo:

Reply:
+1. The Google style guide also explicitly mentions that pragmas should be avoided, and header guards should be used instead.